bad.php

```diff
@@ -0,0 +1,8 @@
+<?php
+namespace Silly;
+
+class Silly_Thing {
```

Comment:
Class name "Silly_Thing" is not in camel caps format
Opening brace of a class must be on the line after the definition